third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -251,7 +250,6 @@ func resourceComputeInstance() *schema.Resource {
 							Type:        schema.TypeString,
 							Optional:    true,
 							Computed:    true,
-							ForceNew:    true,
 							Description: `The project in which the subnetwork belongs.`,
 						},
 
```

Comment:
Doesn't this feature also make the `network_ip` updatable when you move the instance across networks/subnetworks, so you can specify the IP under the new subnet? I recollect testing this and IIRC it seems to be the case.

Reply:
Yes this is true but quite complex to implement as network_ip cannot change under other circumstances. This will require migrating to a customized diff implementation. I will follow-up with this additional change in a subsequent pr

Reply:
If a user has `network_ip` set and they try to update to an incompatible subnetwork/network will they be stuck? Just asking in case this should wait to be merged along with the follow-up.

Reply:
If the user changes subnetwork and does define the network_ip it is inferred